src/test/resources/report.csv

```diff
@@ -0,0 +1 @@
+Hello, world!
```

Comment:
Stop sign means there is no empty line, please do not forget about it.
![image](https://github.com/mate-academy/jv-fruit-shop-tests/assets/82539553/15ebc041-b428-4186-8e4f-5d5998f73052)